test/tests/meshgenerators/generated_mesh_generator/tests

```diff
@@ -102,4 +102,17 @@
     issues = '#13959'
     mesh_mode = 'REPLICATED'
   []
+
+  [with_subdomain_ids_test]
+    type = 'Exodiff'
+    input = 'gmg_with_subdomain.i'
+    cli_args = '--mesh-only'
+    exodiff = 'gmg_with_subdomain_in.e'
+    exodiff_opts = '-pedantic'
+    requirement = 'MOOSE shall be able to assign subdomain IDs fore the regular Cartesian mesh generator.'
+    design = 'meshgenerators/GeneratedMeshGenerator.md'
+    issues = '#19297'
+    mesh_mode = 'REPLICATED'
```

Comment:
I did not remove `mesh_mode = 'REPLICATED'` here because all other tests here are having it. I do not think it is needed. Maybe because those tests are all mesh-only, so no point to run them in distributed mesh mode I guess.

Reply:
Please remove it. If it doesn’t work with distributed mesh, put an error in the object. Otherwise it should be tested with the full suite.